docs/content/preview/explore/change-data-capture/debezium-connector-yugabytedb.md

```diff
@@ -1018,7 +1105,100 @@ Advanced connector configuration properties:
+| auto.add.new.tables | `true` | Controls whether the connector should keep polling the server to check if any new table has been added to the configured change data stream ID. If a new table has been found in the stream ID and if it has been included in the `table.include.list`, the connector will be restarted automatically. |
+| new.table.poll.interval.ms | 300000 | The interval at which the poller thread will poll the server to check if there are any new tables in the configured change data stream ID. |
+
+### Monitoring status of the deployed connector
+
+You can use the rest APIs to monitor your deployed connectors.
+
+1. Get the list of all connectors
+
+   ```sh
+   curl -X GET localhost:8083/connectors/
+   ```
+
+2. Get the configuration of a connector
+
+   ```sh
+   curl -X GET localhost:8083/connectors/<connector-name>
+   ```
+
+3. Get the list of tasks with their configuration
+
+   ```sh
+   curl -X GET localhost:8083/connectors/<connector-name>/tasks
+   ```
+
+   Additionally, you can also view the status of one task by providing its task ID:
+
+   ```sh
+   curl -X GET localhost:8083/connectors/<connector-name>/tasks/<task-id>
+   ```
+
+4. Get the status of the connector with the status of its tasks
+
+   ```sh
+   curl -X GET localhost:8083/connectors/<connector-name>/status
+   ```
+
+## Monitoring
+
+The YugabyteDB source connector provides two types of metrics that are in addition to the built-in support for JMX metrics that Zookeeper, Kafka, and Kafka Connect provide.
+
+* Snapshot metrics provide information about connector operation while performing a snapshot.
+* Streaming metrics provide information about connector operation when the connector is capturing changes and streaming change event records.
+
+### Snapshot metrics
+
+The **MBean** is `debezium.yugabytedb:type=connector-metrics,server=<database.server.name>,task=<task.id>,context=snapshot`.
+
+Snapshot metrics are not exposed unless a snapshot operation is active, or if a snapshot has occurred since the last connector start. The following table lists the shapshot metrics that are available.
+
+| Metric name | Type | Description |
+| :---- | :---- | :---- |
+| LastEvent | `string` | The last snapshot event that the connector has read. |
+| MilliSecondsSinceLastEvent | `long` | The number of milliseconds since the connector has read and processed the most recent event. |
+| TotalNumberOfEventsSeen | `long` | The total number of events that this connector has seen since last started or reset. |
```

Comment:
Just making this match the equivalent in the next section...
```suggestion
| TotalNumberOfEventsSeen | `long` | The total number of events that this connector has seen since the last start or metrics reset. |
```